http/src/androidMain/kotlin/com/mirego/trikot/http/android/requestFactory/KtorHttpRequestFactory.kt

```diff
@@ -24,23 +25,33 @@ import io.ktor.content.TextContent
 import io.ktor.http.ContentType
 import io.ktor.http.HttpMethod
 import io.ktor.util.flattenEntries
+import kotlin.coroutines.CoroutineContext
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.launch
 import org.reactivestreams.Publisher
-import kotlin.coroutines.CoroutineContext
+
+private const val DEFAULT_TIMEOUT_MS = 10000L
 
 class KtorHttpRequestFactory(
     httpLogLevel: LogLevel = LogLevel.NONE,
     httpLogger: Logger = Logger.DEFAULT,
-    private var httpClient: HttpClient = HttpClient()
+    private var httpClient: HttpClient = HttpClient(),
+    requestTimeoutMs: Long = DEFAULT_TIMEOUT_MS,
```

Comment:
That would be nice if we could force passing a `Duration` object here and use `toMilliseconds()` we setting the actual Ktor.HttpTimeout values.

Reply:
I did that first, but Duration is an experimentalApi. To we really want to force client to have the @ExperimentalTime when calling this method?

Reply:
I see that we already use it at many places in Trikot. I'll make the change.